From 37aba887d7494a8d970a860e190bd77765d6b440 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Thu, 22 Dec 2011 19:12:14 +0000
Subject: [PATCH] tests/protocol: Contains a Perl test so we must set
 PERL5LIB.

This fixes commit 498758faee6be7e989869bf7feba5e1026a1bb65.
---
 tests/protocol/Makefile.am |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/tests/protocol/Makefile.am b/tests/protocol/Makefile.am
index 0d899dd..ce9282e 100644
--- a/tests/protocol/Makefile.am
+++ b/tests/protocol/Makefile.am
@@ -32,7 +32,8 @@ TESTS_ENVIRONMENT = \
 	MALLOC_PERTURB_=$(random_val) \
 	LD_LIBRARY_PATH=$(top_builddir)/src/.libs \
 	LIBGUESTFS_PATH=$(top_builddir)/appliance \
-	TMPDIR=$(top_builddir)
+	TMPDIR=$(top_builddir) \
+	PERL5LIB=$(top_builddir)/perl/blib/lib:$(top_builddir)/perl/blib/arch
 
 EXTRA_DIST = \
 	$(TESTS)
-- 
1.7.6

